src/execute-pr-actions.ts

```diff
@@ -153,6 +157,26 @@ async function getMutationsForComments(actions: Actions, pr: PR_repository_pullR
   return mutations;
 }
 
+
+async function getMutationsForCommentRemovals(actions: Actions, pr: PR_repository_pullRequest) {
+  const mutations: Mutation[] = [];
+
+  const travisMessageToKeep = actions.responseComments.find(c => c.tag.startsWith("travis-complaint"))
+  const botComments = (pr.comments.nodes ?? []).filter(comment => comment?.author?.login === "typescript-bot")
+  for (const comment of botComments) {
+    if (!comment) continue
+
+    const parsed = parseComment(comment.body);    
+    if (!parsed) continue
+
+    if (parsed.tag.includes("travis") && parsed.tag !== travisMessageToKeep?.tag) {
+      mutations.push( createMutation(deleteComment, { input: { id: comment.id } }) )
+    }
+  }
+
+  return mutations;
+}
+
```

Comment:
Basically find all bot comments about travis which aren't the same as a potential message we might be posting